tests/lib/util/apply-disable-directives.js

```diff
@@ -371,19 +388,6 @@ describe("apply-disable-directives", () => {
                 []
             );
         });
-
-        it("keeps problems on the next line if there is an eslint-enable comment before the problem on the next line", () => {
-            assert.deepEqual(
-                applyDisableDirectives({
```

Comment:
why did you remove this test?

Reply:
If we're handling `eslint-disable` and `eslint-enable` comments separately from `eslint-disable-line` and `eslint-disable-next-line`, then it doesn't make sense for `eslint-enable` comments to affect the next line.

That test was added as part of https://github.com/eslint/eslint/commit/a32ec3620bf9aa8fae0c7ae94163f3e4a732b3ce because it was the previous behavior before the refactor, but I don't know of anyone relying on it (and it's undocumented anyway).

Reply:
This test gave me some hard time too.
But I think the behavior defined inside this test does make sense. I dont think we should remove it.